core/src/test/java/org/apache/calcite/util/SourceTest.java

```diff
@@ -67,6 +70,19 @@ private static String getRootPrefix() {
     }
   }
 
+  @Test public void testFileToUrl() throws URISyntaxException {
+    final String path = "abc.txt";
+    Source fileSource = file(null, path);
+    URL url = fileSource.url();
+
+    assertFalse(url == null, "No url generated for file source");
```

Comment:
What does this mean?
Could you please remove `== null` from within assert call?

Reply:
Thank you for the good suggestion.
I have revised the code accordingly. Please take anotehr look.